opm/input/eclipse/Schedule/MSW/Compsegs.cpp

```diff
@@ -308,16 +308,12 @@ namespace {
     }
 } // end namespace
 
-
+    
```

Comment:
https://github.com/OPM/opm-common/blob/master/.pre-commit-config.yaml

This PR introduces some white spaces, maybe you can use `pre-commit` to remove them.